### PR TITLE
Core working group notes from 2021-02-26

### PR DESCRIPTION
**Pull request by lschuermann**
### Pull Request Overview

Add core team call notes of 2021-02-26. As always, I hope to have attributed each argument correctly and not misrepresented anyone's opinion.

### Testing Strategy

This pull request was tested by reading with spellcheck on.

### TODO or Help Wanted

N/A

### Documentation Updated

- [x] Updated the relevant files in `/docs`, or no updates are required.

### Formatting

- [N/A] Ran `make prepush`.

### CODE REVIEW

**Comment by lschuermann** [comment]
I'm really sorry this took so long. Due to the pandemic I didn't have access to my workstation where the notes were stored until now. The next ones will be more timely.

File: doc/wg/core/notes/core-notes-2021-02-26.md [hunk 1]
```diff
@@ -0,0 +1,522 @@
...
+* Branded: I'm wondering if there is a way to do option 4, without a
+  big diff. Couldn't we cut off the last few lines of `reset_handler`
+  and have what calls `reset_handler` also call some extra function?
+
+  So we would essentially move the scheduler call into its own
+  function.
+
+* Hudson: Is `reset_handler` called from assembly?
+
+` Brad: No, on Cortex-M platforms it's called by the hardware. On
```
Comment (hudson-ayers) [inline doc/wg/core/notes/core-notes-2021-02-26.md:294]:
```suggestion
* Brad: No, on Cortex-M platforms it's called by the hardware. On
```

**Review by hudson-ayers — APPROVED** [review]

**Review by jrvanwhy — DISMISSED** [review]

**Comment by lschuermann** [comment]
Thanks for the applying the suggestion. Would've squashed the commit but I suppose this is fine.

**Review by hudson-ayers — APPROVED** [review]

**Comment by hudson-ayers** [comment]
bors won't merge without an approving review.

bors r+

**Pull request merged**